Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml

```
@@ -7,8 +7,8 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 50 # DeltaV - Damage Bumped, was 40
-        Structural: 200 # DeltaV - was 30
-    ignoreResistances: true # DeltaV - Hristov be spooky now
+        Piercing: 70 #AntiMateriel Sniper should hurt real bad
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)